data/json/itemgroups/clothing.json

```diff
@@ -188,5 +188,15 @@
       [ "pocketwatch", 25 ],
       [ "wristwatch", 100 ]
     ]
+  },
```

Comment:
`clothing_work_set` could also be found in the garage lockers as per your previous suggestions?